scene/gui/graph_edit.cpp

```diff
@@ -1259,6 +1259,11 @@ HBoxContainer *GraphEdit::get_zoom_hbox() {
 	return zoom_hb;
 }
 
+void GraphEdit::set_controls_visibility(bool visible)
+{
```

Comment:
Two style issues:
- We prefix function parameters with `p_`, so it should be `p_visible`.
- The opening brace should be on the first line.